examples/secp256k1/README.md

```diff
@@ -39,19 +43,41 @@ Assuming you've your local environment set up, let's go to the spore secp256k1 e
 cd examples/secp256k1
 ```
 
-The example code is stored in the [apis directory](./apis), feel free to review the code or change the data of spore/cluster you want to create on-chain, or even treat the examples as your experimental fields.
+The example code is stored in the [examples/secp256k1/apis](./apis) directory. Feel free to review and change anything you want, treating the examples as your experimental playground or a code sandbox to explore and play around with.
 
 To run an example, for instance if you want to create a spore on-chain, you can run the [apis/createSpore.ts](./apis/createSpore.ts) example in your terminal:
```

Comment:
For instance, if you want to create a spore on-chain, run the [apis/createSpore.ts](./apis/createSpore.ts) example in your terminal: